app/scripts/services/mobileClients.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("MobileClientsService", function(DataService,
+                                            Logger) {
+
+    var mobileclientVersion = {
+      group: "mobile.k8s.io",
+      version: "v1alpha1",
+      resource: "mobileclients"
+    };
+
+
+    var watch = function(context, callback, opts ) {
+      return DataService.watch(mobileclientVersion, context, callback, opts);
+    };
+
+    var getMobileClients = function(namespace) {
+      return DataService.list(mobileclientVersion, {namespace: namespace})
+        .then(function(clients) {
+          return clients.by("metadata.name");
+        })
+        .catch(function(error) {
+          Logger.error('MobileClientsService getMobileClients', error);
+        });
+    };
+
+
+    var getClientsWhereExcluded = function(serviceName, mobileClients) {
+      return _.filter(mobileClients, function(client) {
+        return _.includes(client.spec.excludedServices, serviceName);
+      });
+    };
+
+    var canAddClients  = function(serviceName, mobileClients) {
```

Comment:
If this is just wrapping this function I am not sure it is necessary. `filtereExcluded(serviceName, mobileClients).length` gives a falsy value